### PR TITLE
Change New, import, export wizard constructors

### PR DESCRIPTION
**Pull request by rawagner**
(shellText, wizardCategory, wizardName)
(shellText, Stringp[ wizardPath)

### CODE REVIEW

File: plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/topmenu/ExportMenuWizard.java [hunk 1]
```diff
@@ -26,25 +26,36 @@
...
+	protected String[] wizardPath;
 	
 	/**
-	 * Constructor set path to the specific item in new wizard dialog.
-	 * @param text dialog text
-	 * @param path to the specific item in new wizard dialog
+	 * Constructs new export Wizard that can be found in ImportExportWizard.
+	 * @param shellText export wizard text when next is clicked
+	 * @param wizardCategory export wizard category
+	 * @param wizardName export wizard name
```
Comment (odockal) [inline plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/topmenu/ExportMenuWizard.java]: Would be nice to have examples of wizard category in comment. So the user knows instantly what to use here.

**Comment by odockal** [comment]
Just an idea, but could it not be possible to have one abstract class that would include constructors and methods (that seems to be the same in every class except method open) and than have one abstract method open that would each class implements by its need? Otherwise it seems ok, @odockal approves this one.

**Comment by rawagner** [comment]
sounds good! fixed

**Comment by odockal** [comment]
Thanks for fixing! Just one more detail compilation failures, javadoc requires "&gt;" when using sign of ">"... I believe.. please, check it.

**Comment by mlabuda** [comment]
The problem is with syntax. "\" is not the escape char. To use > and < you should use &gt; and &lt;

**Comment by rawagner** [comment]
@odockal fixed

**Pull request merged**